agent/deployment_stages/sensu_healthchecks.py

```diff
@@ -51,13 +51,22 @@ def find_sensu_plugin(plugin_paths, script_filename):
         return None
 
     @staticmethod
-    def generate_check_definition(check, script_absolute_path, platform, instance_tags, logger):
+    def generate_check_definition(check, script_absolute_path, deployment):
+        platform = deployment.platform
+        instance_tags = deployment.instance_tags
+        logger = deployment.logger
+        deployment_slice = deployment.service.slice
+        if deployment_slice is not None and deployment_slice.lower() == 'none':
+            deployment_slice = None
+
         def get_command():
             if platform == 'windows':
                 command = '{0} "{1}"'.format('powershell.exe -NonInteractive -NoProfile -ExecutionPolicy Bypass -file', script_absolute_path)
             else:
                 command = script_absolute_path
-            return '{0} {1}'.format(command, check.get('script_arguments', '')).rstrip()
+            script_args = check.get('script_arguments')
+            script_args = ' '.join(filter(None, (script_args, deployment_slice)))
+            return '{0} {1}'.format(command, script_args).rstrip()
```

Comment:
Have you tried running this with various Sensu plugins, i.e. when we're using `server_script` rather than `local_script`?

For example, we're using https://github.com/sensu-plugins/sensu-plugins-aws/blob/master/bin/check-sqs-messages.rb to monitor queue sizes, and I'm wondering if adding this extra parameter to the script will cause any issues, e.g. options not recognized/change the behavior of the check.